README.md

```diff
@@ -1,5 +1,3 @@
-:rotating_light: **Become part of our Conference on Adaptive Mesh Refinement and Applications (AMR25) on September 02-04, 2025!** The abstract submission deadline is **April 15, 2025**. Visit the [AMR25 website](https://dlr.de/AMR25) for further information. :rotating_light:
```

Comment:
We could still inform about the Conference in our readme, so you could still register as a participant.

Reply:
Good idea 👍